client/settings/deposits/index.js

```diff
@@ -208,6 +210,17 @@ const Deposits = () => {
 		accountStatus: { accountLink },
 	} = useContext( WCPaySettingsContext );
 
+	const { account, overview } = useSelectedCurrencyOverview();
+	const selectedCurrency =
+		overview?.currency || wcpaySettings.accountDefaultCurrency;
+
+	const hasErroredExternalAccount =
+		account?.default_external_accounts?.some(
+			( externalAccount ) =>
+				externalAccount.currency === selectedCurrency &&
+				externalAccount.status === 'errored'
+		) ?? false;
```

Comment:
💭 I found that when I used a multi-deposit-currency store, this notice did not display, and there is no way to see which currency is selected on the settings page.

What do you think about making this display if _any_ currency's default deposit account is in an `errored` state?

```suggestion
	const { overviews } = useAllDepositsOverviews();

	const hasErroredExternalAccount =
		overviews.account?.default_external_accounts?.some(
			( externalAccount ) => externalAccount.status === 'errored'
		) ?? false;
```

Reply:
>What do you think about making this display if any deposit currency's default deposit account is in an errored state?

Thanks for testing this with a multi-currency account! what you shared here makes sense!

Reply:
I have incorporated the feedback here - [e6afbda](https://github.com/Automattic/woocommerce-payments/pull/9482/commits/e6afbda8c99e5ad509af4332e9759731c8a3e07b)

Also updated the tests to include multi-currency default accounts scenario - [4e8a2ba](https://github.com/Automattic/woocommerce-payments/pull/9482/commits/4e8a2ba5c499ec46d8602c708097223dc1e27069)

Thanks once again, for the valuable review, Eric 🙌🏼

Reply:
>I've tested this with a multi-deposit-currency store with a single errored deposit destination.

I did a round of manual test this time with a similar account, and it seems to work fine too